template/README.md

```diff
@@ -1,3 +1,41 @@
 # ${NAME}
 
 ## _A Foxglove Studio Extension_
+
+[Foxglove Studio](https://github.com/foxglove/studio) allows developers to create extensions, or custom code that is loaded and executed inside the Foxglove Studio application. This can be used to add custom panels. Extensions are authored in TypeScript using the `@foxglove/studio` SDK.
+
+## Develop
+
+Extension development uses the `yarn` package manager to install development dependencies and run build scripts.
```

Comment:
does extension development work with npm? I think we should recommend that by default if possible, since it is far better documented across the internet for users unfamiliar with the JS ecosystem (especially considering the yarn 1 vs 2 fiasco).

users who which to use yarn will know what they are doing and how to translate the various commands.

Reply:
switched instructions to npm